src/pages/forget-password/forms/ResetPassForm.js

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useState } from 'react'
+import { Grid, IconButton, InputAdornment } from '@mui/material'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { useForm } from 'src/hooks/form'
+import toast from 'react-hot-toast'
+import * as yup from 'yup'
+import { AuthContext } from 'src/providers/AuthContext'
+import { useAuth } from 'src/hooks/useAuth'
+import { useError } from 'src/error'
+import NewPassword from 'src/components/Shared/NewPassword'
+import axios from 'axios'
+import { useRouter } from 'next/router'
+
+const ResetPassForm = ({ labels, username = '' }) => {
+  const [score, setScore] = useState(0)
+  const { stack: stackError } = useError()
+  const auth = useAuth()
+  const { encryptePWD } = useContext(AuthContext)
+  const router = useRouter()
+
+  const { formik } = useForm({
+    enableReinitialize: true,
+    validateOnChange: true,
+    initialValues: {
+      username: username ? username : auth?.user?.username,
+      code: '',
+      newPassword: '',
+      confirmPassword: ''
+    },
+    validationSchema: yup.object({
+      newPassword: yup.string().required(),
+      confirmPassword: yup.string().required()
+    }),
+    onSubmit: async () => {
+      if (formik.values.newPassword === formik.values.confirmPassword) {
+        const resetPWVal = {
+          userName: formik.values.username,
+          password: encryptePWD(formik.values.newPassword),
+          code: formik.values.code
+        }
+
+        var bodyFormData = new FormData()
+        bodyFormData.append('record', JSON.stringify(resetPWVal))
+
+        axios
+          .post(`${process.env.NEXT_PUBLIC_AuthURL}MA.asmx/resetPW`, bodyFormData)
+          .then(res => {
+            toast.success(labels.passSuccess)
+            router.push('/login')
+          })
+          .catch(error => {
+            stackError({ message: error })
+          })
+      } else {
+        toast.error(labels.passNotMatching)
+      }
+    }
+  })
+
+  return (
+    <VertLayout>
+      <Grow>
+        <Grid container spacing={3} sx={{ pl: '10px', pt: '10px', pr: '10px' }}>
```

Comment:
instead pl: '10px', pr: '10px' use px:'10px'

px is shorthand for setting both pl and pr